### PR TITLE
adding policy-registry

### PR DESCRIPTION
**Pull request by ac-obair**
adding policy register to test subscription

### CODE REVIEW

File: policy-registry/main.tf [hunk 1]
```diff
@@ -0,0 +1,32 @@
+terraform {
+  backend "azurerm" {
+    storage_account_name = "actfstate"
+    container_name       = "ac-iac-platform"
+    key                  = "demo/policy-registry"
+  }
+  required_version = "= 0.12.29"
```
Comment (zoemac11) [inline policy-registry/main.tf:7]: You'll need to use terraform 13 for the azure devops stuff

Comment (ac-obair) [reply policy-registry/main.tf:7]: is 0.13.3 ok or a specific version

File: policy-registry/role-assignment-azure-devops.tf [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+resource azuread_service_principal_password "arnoldclark-azdevops-policy-registry" {
+    service_principal_id = azuread_service_principal.arnoldclark-azdevops-policy-registry.id
+    value                = random_password.arnoldclark-azdevops-policy-registry.result
+    end_date             = "2099-01-01T01:02:03Z"
+}
+
+resource azuredevops_serviceendpoint_azurerm "policy-registry" {
+  project_id            = data.azuredevops_project.dpd.id
+  service_endpoint_name = "policy-registry"
```
Comment (zoemac11) [inline policy-registry/role-assignment-azure-devops.tf:22]: to keep the convention of the rest I would name it something like 'azure-platform-policy-registry'

File: policy-registry/role-assignment-azure-devops.tf [hunk 3]
```diff
@@ -0,0 +1,43 @@
+resource "random_password" "arnoldclark-azdevops-policy-registry" {
+  length = 32
+  special = true
+}
+
+resource azuread_application "arnoldclark-azdevops-policy-registry" {
+  name = "arnoldclark-azdevops-policy-registry"
+}
```
Comment (zoemac11) [inline policy-registry/role-assignment-azure-devops.tf:8]: probably want something in this as well to show its for our test subscription, 'arnoldclark-azdevops-platform-policy-registry' ?

Comment (ac-obair) [reply policy-registry/role-assignment-azure-devops.tf:8]: I'm not sure what you mean there

**Review by zoemac11 — APPROVED** [review]

**Pull request merged**